locales/en-US.json

```diff
@@ -43,6 +43,7 @@
   "discNumber": "Disc {discNumber}",
   "dislikes": "Dislikes",
   "edit": "Edit",
+  "editMetadata": "Edit metadata",
```

Comment:
Tabulation is weird, isn't it?

Reply:
Wdym?

Reply:
Sorry, I was on the phone and all the other lines weren't "tabuled" the same